app/src/features/pricing/components/UpgradeSuccess.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { Card, Result } from "antd";
+import PageLoader from "components/misc/PageLoader";
+import { getUserAuthDetails } from "store/selectors";
+import { getBillingTeamRedirectURL } from "backend/billing";
+import PATHS from "config/constants/sub/paths";
+import { getFunctions, httpsCallable } from "firebase/functions";
+import { getPrettyPlanName } from "utils/FormattingHelper";
+import Logger from "lib/logger";
+
+export const UpgradeSuccess: React.FC = () => {
+  const navigate = useNavigate();
+  const user = useSelector(getUserAuthDetails);
+  const userPlanName = user?.details?.planDetails?.planName;
+
+  useEffect(() => {
+    if (user.loggedIn && userPlanName) {
+      const sendPlanUpgradedNotification = httpsCallable(
+        getFunctions(),
+        "premiumNotifications-sendPlanUpgradedNotification"
+      );
+      sendPlanUpgradedNotification({
+        planName: getPrettyPlanName(userPlanName),
+      }).catch((e) => {
+        Logger.log(e);
+      });
+
+      getBillingTeamRedirectURL(user?.details?.profile?.uid).then((redirectUrl) => {
+        if (!redirectUrl) {
+          navigate(PATHS.SETTINGS.BILLING.RELATIVE);
+        } else {
+          navigate(redirectUrl);
```

Comment:
do we need to add some extra params in the redirecUrl like [here](https://github.com/requestly/requestly/blob/ENGG-2169/app/src/features/pricing/components/PricingModal/Checkout.tsx#L46)

Reply:
No, not needed